variable/technology/electricity grid.yaml

```diff
@@ -0,0 +1,25 @@
+# List of variables related to the electricity grid
```

Comment:
having file names with spaces can create all kinds of problems, so I suggest to rename to `electricity-grid.yaml`

Reply:
@danielhuppmann done